plugins/content/pagebreak/pagebreak.php

```
@@ -127,6 +127,11 @@ public function onContentPrepare($context, &$row, &$params, $page = 0)
 		// Split the text around the plugin.
 		$text = preg_split($regex, $row->text);
 
+		if (!isset($text[$page]))
+		{
+			JError::raiseError(404, JText::_('PLG_CONTENT_PAGEBREAK_PAGE_NOT_FOUND'));
```

JError will be deprecated in 4.0.
See https://api.joomla.org/cms-3/classes/JError.html

Isn't better to throw a PHP exception here. Example:
```php
throw new Exception(JText::_('PLG_CONTENT_PAGEBREAK_PAGE_NOT_FOUND'), 404);
```